src/mqtt/MQTT.cpp

```diff
@@ -902,7 +902,9 @@ std::string MQTT::meshPacketToJson(meshtastic_MeshPacket *mp)
         jsonObj["snr"] = new JSONValue((float)mp->rx_snr);
     if (mp->hop_start != 0 && mp->hop_limit <= mp->hop_start)
         jsonObj["hops_away"] = new JSONValue((unsigned int)(mp->hop_start - mp->hop_limit));
-
+    jsonObj["hop_start"] = new JSONValue((unsigned int)(mp->hop_start));
```

Comment:
I would only add `hop_start` when it's not 0 because that usually means a node on the path had too old firmware. 

Furthermore, we don't need `hop_limit` because it can be derived from `hop_start` and `hops_away`.